transport/cert/secureconnect_cert_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package cert
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+)
+
+func TestSecureConnectSource_ConfigMissing(t *testing.T) {
+	source, err := NewSecureConnectSource("missing.json")
+	if !errors.Is(err, errSourceUnavailable) {
+		t.Fatal("NewSecureConnectSource: expected errSourceUnavailable returned when config is missing.")
```

Comment:
nit: For all testing failures error messages should be in the form got then want.(opposite of many other langs) https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

Reply:
Updated tests to be got then want. Thanks for your review and suggestions Cody!